lib/modules/platform/github/common.ts

```diff
@@ -11,6 +12,7 @@ export function coerceRestPr(pr: GhRestPr | null | undefined): Pr | null {
     return null;
   }
 
+  const bodyStruct = pr.bodyStruct ?? getPrBodyStruct(pr.body);
```

Comment:
Is this meant to convert cached `pr.body` values? And should we *delete* `pr.body` in that case? Or is it already done in compactPrBodyStructure()?

Reply:
It's not necessary to modify `pr` object here, we just don't put `body` to the result